### PR TITLE
fix: dune build

### PR DESCRIPTION
**Pull request by svishnus**
### Dune `grvweb` build fix
- Currently `dune` doesn't copy `index.html` and `style.css` to `_build`
- Fixing this by using `haz3lweb`'s approach of moving all fixed resources to a different folder

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Grove
 
+### Move statics and update `dune` 
+- Move `index.html` and `style.css` and all other `.js` deps to a separate directory
```
Comment (cyrus-) [inline README.md:4]: this isn't needed anymore, should be done by the makefile

File: Makefile [hunk 2]
```diff
@@ -12,6 +14,7 @@ dev:
 	dune build @src/fmt --auto-promote || true
 	dune build --profile dev
 	dune build @doc
+	dune build src/grvweb/{main.bc.js,index.html,style.css,viz.js,lite.render.js}
```
Comment (cyrus-) [inline Makefile:17]: instead of individually naming each file here, put them all in a `www` directory like we do in hazel and move that

**Review by cyrus- — CHANGES_REQUESTED** [review]

File: Makefile (hunk 2 above)
Comment (svishnus) [reply Makefile]: We only need here index.html and style.css here

**Review by svishnus — COMMENTED** [review]
Project should build okay now

**Pull request merged**